views/donations/upload.handlebars

```diff
@@ -0,0 +1,31 @@
+{{#if msg}}
+  {{'hey'}}
+{{else}}
+  {{msg}}
+{{/if}}
+
+
```

Comment:
Do we need this code here?

You have already displayed the message on line no 8. I think we should be okay after removing these lines.